### DISCUSSION TITLE
MultiGPUs support

### DISCUSSION THREAD

**Discussion by imedjendoubi** [lllyasviel/stable-diffusion-webui-forge#1751] [Q&A]
Can I use multigpu to accelerate photo generation?

Thank you!

**Comment by ArnorWing** [comment]
No, you can use seperate installs and have more than 1 instance running. 
You can just clone your current install, no need to make a new one.

**Comment by imedjendoubi** [comment]
I don't want to have two instances, I want to use multiple GPUs for a single instance to accelerate TXT2IMG generation.

I have a multigpus setup with 1 cpu and bunch of RAM

**Comment by aiW2o3l4f** [comment]
I have installed two 3070 graphics cards (8GB) on my computer Although there is a total of 16GB of video memory, it still faces the problem of insufficient memory because SD can only use one graphics card 
I used an old version of Ollma before, which by default could use the video memory of two graphics cards at the same time. After starting Ollma, each graphics card occupied approximately 4Gb of video memory.

**Reply by imedjendoubi** [reply]
Yes exactly, that's what I am trying to do

I have couple of 3070s just laying around.
